packages/descendant/src/use-descendant.ts

```diff
@@ -7,7 +7,7 @@ import { useSafeLayoutEffect, cast } from "./utils"
  * @internal
  * React hook that initializes the DescendantsManager
  */
-function useDescendants<T extends HTMLElement = HTMLElement, K = {}>() {
+function useDescendants<T extends HTMLElement = HTMLElement, K extends Record<string, any> = {}>() {
   const descendants = useRef(new DescendantsManager<T, K>())
```

Comment:
Context: the type `DescendantsManager<T, K>` has a constraint of `Record<string, any>` on `K`, so cannot be instantiated with `null` or `undefined`, but `useDescendants` *can* be instantiated with those types in that position, hence there's a potential bug here without this PR in place.